valohai_cli/models/project.py

```diff
@@ -84,6 +85,11 @@ def get_execution_from_counter(
                 params=(params or {}),
             ).json()
             assert isinstance(data, dict)
+
+            if raise_on_not_running and data["status"] == "queued":
+                raise ExecutionNotRunning(
+                    f"Execution #{counter} is not running. It's current status is: {data['status']}",
```

Comment:
Typo: `it's` -> `its`.
```suggestion
                    f"Execution #{counter} is not running. Its current status is: {data['status']}",
```

Also, you're ensuring `status` is queued above, so there's no way this could ever be "Its current status is: queued"...